stencils/pace/stencils/testing/test_translate.py

```diff
@@ -167,6 +168,46 @@ def process_override(threshold_overrides, testobj, test_name, backend):
             )
 
 
+def perturb(input):
+    roundoff = 1e-16
+    for data in input.values():
+        if isinstance(data, np.ndarray) and data.dtype in (np.float64, np.float32):
+            not_fill_value = data < 1e10
```

Comment:
Is this safe? I'm thinking we don't have anything larger than 1e10, but isn't fill value usually ~1e30?

Reply:
I could use a larger threshold, I should do a (automatic, in code here) search to see first whether any other fill values are used. I thought there were values other than 1e30, sincet the reference Fortran also sometimes uses some random fill values in its code.